readme.md

```diff
@@ -30,6 +38,7 @@ A couple of default styles lifted from [Marked.app][marked].
 
 [mmdql]: https://github.com/fletcher/MMD-QuickLook
 [screenshot]: https://github.com/ttscoff/MMD-QuickLook/raw/master/MMDQLGithub-1.jpg
+[cask]: http://caskroom.io/
```

Comment:
Why? The link works in Marked2 app and also on https://github.com/oschrenk/MMD-QuickLook/tree/homebrew-cask-readme 

According to http://daringfireball.net/projects/markdown/syntax#link the id is the the second bracket pair

```
This is [an example] [id] reference-style link.

Then, anywhere in the document, you define your link label like this, on a line by itself:

[id]: http://example.com/  "Optional Title Here"
```